test/functional/update/slow-server/test.bats

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bats
+
+load "../../swupdlib"
+
+server_pid=""
+port=""
+
+targetfile="6c27df6efcd6fc401ff1bc67c970b83eef115f6473db4fb9d57e5de317eba96e"
+
+setup() {
+  # extra debug for touchy test
```

Comment:
heh, reminds me when we had to debug and fix the test you copied this from. this is a situation it was good you copied the existing test.

Reply:
oops. I'll take that out from my test